src/pages/landing/landing.ts

```diff
@@ -17,7 +27,14 @@ export class LandingPage {
     this.navCtrl.push('RegisterPage');
   }
 
-  openResetPasswordPage(): void {
-    this.navCtrl.push('ResetPasswordPage');
+  openDownloadPage(): void {
+    this.navCtrl.push('DownloadPage');
+  }
+
+  checkIfBrowser(): void {
+    // Determine if user is viewing on desktop
+    if (this.platform.is('core')) {
+      this.isBrowser = true;
```

Comment:
No need for the `if` statement. This can just be simplified to `this.isBrowser = this.platform.is('core')`

Reply:
Smart man 👍will make this update